lib/beaker/hypervisor/openstack.rb

```diff
@@ -63,6 +63,17 @@ def initialize(openstack_hosts, options)
         raise "Unable to create OpenStack Network instance (api_key: #{@options[:openstack_api_key]}, username: #{@options[:openstack_username]}, auth_url: #{@options[:openstack_auth_url]}, tenant: #{@options[:openstack_tenant]})"
       end
 
+      # Validate openstack_volume_support setting value, reset to boolean if passed via ENV value string
+      if not @options[:openstack_volume_support].is_a?(TrueClass) || @options[:openstack_volume_support].is_a?(FalseClass)
+        if @options[:openstack_volume_support].match(/\btrue\b/i)
+          @options[:openstack_volume_support] = true
+        elsif @options[:openstack_volume_support].match(/\bfalse\b/i)
+          @options[:openstack_volume_support] = false
+        else
+          raise "Invalid value provided for CONFIG setting openstack_volume_support, current value #{@options[:openstack_volume_support]}"
+        end
+      end
```

Comment:
@ipcrm this seems like an awful lot of work for what you're trying to get at. Is there some use case where a string or something like that must be passed in, so you can't just rely on this being of type `TrueClass` or `FalseClass`?

We would prefer to rely on people setting the yaml correctly and failing on the case when people have set the wrong type for these. That keeps the code here simple, and in places like hypervisors or parsers, it's not a lot of wasted time to fail since things are just getting started.

also, it seems to be the case that if someone passed in a `FalseClass` `false` value, this wouldn't set the `@options[:openstack_volume_support]` value correctly, would it? I would imagine the outer if statement would need an else case to take care of that, no?